### ISSUE TITLE
Animal Crossing: New Horizons softlocks on workbench customization

### ISSUE THREAD

**Issue by goldenx86** [yuzu-emu/yuzu#4821]
Any game version is affected.
Affects AMD and Intel GPUs, both in OpenGL and Vulkan. Seems to also affect Nvidia on Vulkan.
[yuzu_log.zip](https://github.com/yuzu-emu/yuzu/files/5428137/yuzu_log.zip)

**Comment by lat9nq** [comment]
It crashes here both on NVIDIA Vulkan as well as on RadeonSI. Tested with ACNH 1.5.1

NVIDIA Vulkan:
[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/5428337/yuzu_log.txt)

RadeonSI:
[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/5428397/yuzu_log.txt)

**Comment by DanMossa** [comment]
I just made this ticket, I'm unsure if this is the same issue

https://github.com/yuzu-emu/yuzu/issues/4823

**Comment by choppymaster** [comment]
SaveReportWithUser Hey @goldenx86 and @lat9nq , I observed 2 missing HLE service commands from your logs. One is "SaveReportWithUser" and another is "RequestImmediateTransmission" . The port of these HLE commands is "prepo:u . I observed the first missing command on the log of issue #5006. I think this issue and issue #5006 is caused by these missing commands. I opened an issue about the SaveReportWithUser, but that issue was closed by German77. Can I add the second command and how about reopen that issue? Hope this will help you guys. Thank you.
